src/components/submission/metadata-row.vue

```diff
@@ -167,7 +171,7 @@ export default {
   }
 
   .icon-angle-right {
-    bottom: #{$padding-right-table-data + 1px};
+    bottom: #{$padding-bottom-table-data + 1px};
```

Comment:
I'm pretty sure this was a typo. `$padding-right-table-data` and `$padding-bottom-table-data` have the same value, so there's no functional change.